src/Exceptions/Handler.php

```diff
@@ -1,22 +1,50 @@
 <?php namespace Orchestra\Testbench\Exceptions;
 
 use Exception;
+use Illuminate\Foundation\Testing\HttpException;
+use Illuminate\Auth\Access\AuthorizationException;
+use Illuminate\Contracts\Validation\ValidationException;
```

Comment:
Should have been `Illuminate\Validation\ValidationException`, and we need to bump `laravel/framework` version constraint to `~5.2.7`.